packages/mksh.rb

```diff
@@ -0,0 +1,31 @@
+require 'package'
+
+class Mksh < Package
+  description 'The MirBSD Korn Shell'
+  homepage 'https://www.mirbsd.org/mksh.htm'
+  version '0.57'
+  source_url 'http://www.mirbsd.org/MirOS/dist/mir/mksh/mksh-R57.tgz'
+  source_sha256 '3d101154182d52ae54ef26e1360c95bc89c929d28859d378cc1c84f3439dbe75'
+
+  depends_on 'make'
+  depends_on 'manpages'
+
+  def self.build
+    system 'sh Build.sh'
+  end
+
+  def self.check
+    system 'sh test.sh'
+  end
+
+  def self.install
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/bin"
+    system "install -c -s -m 555 mksh #{CREW_DEST_PREFIX}/bin/mksh"
+#    Can't perform this step unless the filesystem is mounted as RW:
+#    system 'grep -x /bin/mksh /etc/shells >/dev/null || echo /bin/mksh >>/etc/shells'
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/share/doc/mksh/examples"
+    system "install -c -m 444 dot.mkshrc #{CREW_DEST_PREFIX}/share/doc/mksh/examples/"
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/share/man/man1"
+    system "install -c -m 444 lksh.1 mksh.1 #{CREW_DEST_PREFIX}/share/man/man1/"
+  end
```

Comment:
Might want to add a `dep self.postinstall` section to explain how to use the shell.  Something like:
```
puts "To use mksh, copy #{CREW_PREFIX}/share/doc/mksh/examples/dot.mkshrc to #{HOME}/.mkshrc".lightblue
```

Reply:
added this to postinstall now, also informed of how they won't be able to use it as the default shell.

i'm personally currently using it by calling `mksh -l` at the end of my ~/.bashrc, but since i'm not sure whether this would be recommended behaviour, or of any adverse consequences i haven't included it in the postinstall message (yet)

Reply:
That would work until you type `exit` from mksh and return to bash.

Reply:
yeah that's true... though i've just checked my configs and remembered i set mksh as my run command for tmux instead. so all my new tmux tabs get started with mksh and `exit` works as expected :)